### PR TITLE
Updated camel caption for ottojsutter file

### PR DESCRIPTION
**Pull request by ottojsutter**
Related to #2 

@a-a-ron what do you think?

### CODE REVIEW

**Comment by githubteacher** [comment]
Global conversation level

File: _posts/0000-01-09-ottojsutter.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![humpday](https://cloud.githubusercontent.com/assets/16547949/25400910/10641d9e-29c2-11e7-80c8-5c38ea261b22.jpg)
 {: .center}
 
-Caption here
+Camel master
```
Comment (githubteacher) [inline _posts/0000-01-09-ottojsutter.md]: Talk about the change that they made on this line

👍 🚢

Comment (baustin612) [inline _posts/0000-01-09-ottojsutter.md]: Test

File: _posts/0000-01-09-ottojsutter.md [hunk 2]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![humpday](https://cloud.githubusercontent.com/assets/16547949/25400910/10641d9e-29c2-11e7-80c8-5c38ea261b22.jpg)
```
Comment (githubteacher) [inline _posts/0000-01-09-ottojsutter.md]: For the sake of demonstration

Comment (baustin612) [inline _posts/0000-01-09-ottojsutter.md]: Testing again

**Pull request merged**